readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -40,9 +40,16 @@ class BaseSphinx(BaseBuilder):
 
     sphinx_doctrees_dir = "_build/doctrees"
 
+    # Output directory relative to where the repository was cloned
+    # (e.g. "_readthedocs/<format>")
+    relative_output_dir = ""
```

Comment:
Empty string feels like an odd default. I think we probably want `None` to make it blow up when not set properly?

Reply:
Yeap. I think I pushed this by mistake while testing why one of the tests was failing. Thanks for pointing it!